### PR TITLE
Adding sample pattern files for linkerd and osm testing

### PR DESCRIPTION
**Pull request by hershd23**
Signed-off-by: hershd23 <hershdhillon23@gmail.com>

**Description**

This PR adds the pattern files required for https://github.com/layer5io/meshery-smp-action/issues/57

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by hershd23** [comment]
@gyohuangxin

File: samples/bookstore.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+name: Autogenerated
+services:
+  7b24589e-2602-46d2-b82e-c0df9d3618d4:
```
Comment (gyohuangxin) [inline samples/bookstore.yaml:3]: Please correct if I'm wrong. Where does these random strings come from? Following this example: https://github.com/service-mesh-patterns/service-mesh-patterns/blob/master/samples/bookInfoPattern.yaml, it seems we should set a fix name here.

File: samples/bookstore.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+name: Autogenerated
```
Comment (gyohuangxin) [inline samples/bookstore.yaml:1]: It's interesting that the pattern file can be autogenerated. What's the difference between `Autogenerated` and [`MesheryGeneratedPatternFile`](https://github.com/service-mesh-patterns/service-mesh-patterns/blob/master/samples/bookInfoPattern.yaml#L1) ?

Comment (hershd23) [reply samples/bookstore.yaml:1]: I am pretty sure this is meshery generated, so not really sure about the difference

File: samples/bookstore.yaml (hunk 1 above)
Comment (hershd23) [reply samples/bookstore.yaml:3]: These random strings come from piecing together the k8s manifests into a pattern file. I think the logic is to simply replace any missing names with a random alphanumeric string

**Comment by hershd23** [comment]
The fact that I had these pattern files up on some branch on github was enough for me to get the tests running.

```

▸LINKERD
Use the arrow keys to navigate:↓↑→←
? Select a Service Mesh from the list: 
▸LINKERD
Use the arrow keys to navigate:↓↑→←
? Select a Service Mesh from the list: 
▸LINKERD
✔LINKERD
25hOnboarding application... Standby for few minutes...
invalid workload of type: Service.K8s
Service Mesh: Linkerd - LINKERD
Endpoint URL: http://localhost:8080
```

Got an invalid workload error, by a pattern file made by meshery. Seems weird @gyohuangxin @Revolyssup

**Comment by Revolyssup** [comment]
> 

This invalid workload error is due to the fact that kubernetes components weren't being registered because that only happens when user sends a request. A hack I would suggest is to run `pattern apply`(this will fail but send a request to server) and then sleep 30-40 seconds.  Then try again and it will succeed definitely.

**Comment by Revolyssup** [comment]
> > 
> 
> This invalid workload error is due to the fact that kubernetes components weren't being registered because that only happens when user sends a request. A hack I would suggest is to run `pattern apply`(this will fail but send a request to server) and then sleep 30-40 seconds. Then try again and it will succeed definitely.

@hershd23

**Pull request merged**